### PR TITLE
Change name and version for aws charts

### PR DESCRIPTION
**Pull request by anushkamittal2001**
Change name for aws chart to provide a Helm chart uri that follows the Semver2 format.

### CODE REVIEW

File: aws/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
-name: kyverno-chart
-version:  v1.12.0-04-aws
+name: kyverno-aws-chart
+version:  v1.12.1
```
Comment (pns-nirmata) [inline aws/Chart.yaml:4]: There are a couple of places in scripts folder with this version. Need to fix that too.

**Review by pns-nirmata — APPROVED** [review]
Merge after handling changes.

File: aws/Chart.yaml (hunk 1 above)
Comment (anushkamittal2001) [reply aws/Chart.yaml:4]: Got it.

File: charts/enterprise-kyverno-operator/Chart.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.3.10
-appVersion: v0.2.2
+version: v0.3.11
+appVersion: v0.2.3
```
Comment (pns-nirmata) [inline charts/enterprise-kyverno-operator/Chart.yaml:6]: Not seeing a 0.2.3 version in kyverno-operator yet.

**Review by pns-nirmata — APPROVED** [review]

File: charts/enterprise-kyverno-operator/Chart.yaml (hunk 2 above)
Comment (anushkamittal2001) [reply charts/enterprise-kyverno-operator/Chart.yaml]: Looks like resolving issues in the operator might take time. Going back to the old image for now.

**Pull request merged**